wiki/Installation/vi.md

```diff
@@ -0,0 +1,77 @@
+# Cài đặt
+
+*Xem thêm: [Cài đặt trên macOS](/wiki/Installation/macOS)*
+
+Trang này sẽ hướng dẫn bạn làm thế nào để làm osu! hoạt động trên thiết bị Windows của bạn. [osu!academy](/wiki/osu!academy) đã có một [video hướng dẫn](https://www.youtube.com/watch?v=0V5GwzmMhpU) làm thế nào để cài đặt osu! trên Windows.
+
+## Yêu cầu tối thiểu 
+
+- .NET framework 4+ (bộ cài thường sẽ nhắc bạn để cài đặt nó, nếu không thì bạn có thể [tải nó tại](https://www.microsoft.com/en-us/download/details.aspx?id=48130)).
```

Comment:
```suggestion
- .NET framework 4.5.2+ (bộ cài thường sẽ nhắc bạn để cài đặt nó, nếu không thì bạn có thể [tải nó tại đây](https://dotnet.microsoft.com/download/dotnet-framework)).
```
This should be changed in all translation files. It's quite outdated.
Although osu! only needs 4.0 to run, but in order to run "osu! updater" flawlessly, which uses TLS 1.2 protocol, you'll need `lib32-gnutls` (on Linux) and at least .NET Framework 4.5 (on Linux) for it to work (and not encountering update loop or "no internet connection")
That's also why Windows XP users can't play osu! anymore (actually they can, but can't install nor update, they'll have to get a copy from another computer)
So I suggest using a version that is still maintained (haven't reached "End of life"), which are 4.5.2 (supports end at Apr 26, 2022), or 4.6.2 (latest version that supports Windows 7).

Reply:
Since folk on discord said to put it as it is, I'll mark this as resolve.